scene/main/node.cpp

```diff
@@ -2189,7 +2189,7 @@ NodePath Node::get_path_to(const Node *p_node, bool p_use_unique_path) const {
 		common_parent = common_parent->data.parent;
 	}
 
-	ERR_FAIL_NULL_V(common_parent, NodePath()); //nodes not in the same tree
+	ERR_FAIL_NULL_V_MSG(common_parent, NodePath(), "Nodes do not share a common parent, make sure they are in the same tree."); // Nodes not in the same tree.
```

Comment:
The comment could be dropped as it does not provide any additional information anymore.